File: app/src/main/java/com/miandroidchallenge/ucoppp/miandroidchallenge/ui/deliverylistfragment/interfaces/OnDeliveryCallback.java
package com.miandroidchallenge.ucoppp.miandroidchallenge.ui.deliverylistfragment.interfaces;

import com.miandroidchallenge.ucoppp.miandroidchallenge.models.DeliveriesModel;

public interface OnDeliveryCallback {

    void onClickDelivery(DeliveriesModel delivery);
    void onFailCallDelivery();
}

File: app/src/main/java/com/miandroidchallenge/ucoppp/miandroidchallenge/ui/deliverylistfragment/interfaces/OnDeliveryCallback.kt
package com.miandroidchallenge.ucoppp.miandroidchallenge.ui.deliverylistfragment.interfaces

import com.miandroidchallenge.ucoppp.miandroidchallenge.models.DeliveriesModel

interface OnDeliveryCallback {
    fun onClickDelivery(delivery: DeliveriesModel)
    fun onFailCallDelivery()
}